_includes/api/en/5x/req-host.md

```diff
@@ -0,0 +1,24 @@
+<h3 id='req.host'>req.host</h3>
+
+Contains the hostname and port number, concatenated
+with a colon, derived from the `Host` HTTP header.
+IPv6 addresses will be wrapped in square brackets as well.
```

Comment:
I realized that I gave two options before, and after looking at which one you picked, it turns out that the wording I gave was wrong 😢 The issue here is that most of the time, `req.host` is never going to actually contain the port number, and this will now be misleading as it seems to say it would indeed contain the port number. Maybe just going with the first one will work?

```suggestion
Contains the host derived from the `Host` HTTP header.
```

Reply:
@dougwilson - done, compressed as per your suggestion.